<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>thresholddlg</class>
 <widget class="QDialog" name="thresholddlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Min</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="min_bar_slide">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
         <property name="tickInterval">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="min_value_spin">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Max</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="max_bar_slide">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="max_value_spin">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Threshold setting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="threshold_combo">
         <item>
          <property name="text">
           <string>THRESH_BINARY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>THRESH_BINARY_INV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>THRESH_TRUNC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>THRESH_TOZERO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>THRESH_TOZERO_INV</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>thresholddlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max_bar_slide</sender>
   <signal>valueChanged(int)</signal>
   <receiver>max_value_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max_value_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>max_bar_slide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min_value_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>min_bar_slide</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min_bar_slide</sender>
   <signal>valueChanged(int)</signal>
   <receiver>min_value_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>thresholddlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
